lib/aur-search

```diff
@@ -9,63 +9,51 @@ query_type=search
 search_by=name-desc
 sort_key=Name
 
-json_short() {
-    local Name Version NumVotes Maintainer OutOfDate Description
+parse_short() {
+    jq -r --arg key "$1" '[.results[]] | sort_by(.[$key])[] | [
+        .Name         // "-",
```

Comment:
unrelated change (and `//` only makes sense on fields that might be null or empty, never the case for name, version e.a.)

Reply:
Is it documented somewhere which exactly fields are optional and which are required? I was hoping it would be documented [here](https://wiki.archlinux.org/index.php/AurJson), but it doesn't seem to be the case.

Reply:
I don't think so, unless you go investigate the PHP code! :grimacing:  What was there before is purely empirical

Reply:
Then I see even less reasons to remove the default that saves us from potentially missing values... But I pushed this nevertheless, used the empirical data from `master` 🙂

Reply:
This is done